modules/oauth20/cms_oauth_20_workflow.e

```diff
@@ -20,7 +20,7 @@ feature {NONE} -- Initialization
 		do
 			initilize (a_consumer)
 			create config.make_default (a_consumer.api_key, a_consumer.api_secret)
-			config.set_callback (a_host + "/account/"+ a_consumer.callback_name)
+			config.set_callback (a_host + "/account/oauth-callback"+ a_consumer.callback_name)
```

Comment:
For me .. a "name" should not start with a slash


On Tue, Jun 30, 2015 at 10:53 PM, Javier Velilla <notifications@github.com>
wrote:

> In modules/oauth20/cms_oauth_20_workflow.e
> <https://github.com/EiffelWebFramework/ROC/pull/32#discussion_r33622452>:
>
> > @@ -20,7 +20,7 @@ feature {NONE} -- Initialization
> >  		do
> >  			initilize (a_consumer)
> >  			create config.make_default (a_consumer.api_key, a_consumer.api_secret)
> > -			config.set_callback (a_host + "/account/"+ a_consumer.callback_name)
> > +			config.set_callback (a_host + "/account/oauth-callback"+ a_consumer.callback_name)
>
> it depends on how we configure the callback_name, but we can add / after
> callback and the consumer callback name does no need to add /.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/EiffelWebFramework/ROC/pull/32/files#r33622452>.
>